### DISCUSSION TITLE
Configuring two different OTBRs on 2 devices

### DISCUSSION THREAD

**Discussion by muralidhar-bn** [openthread/openthread#10668] [Q&A]
I would like to configure two different OTBR's on two Raspberry Pi 4 for creating two different Thread networks. I comissionned both OTBR's using two different controllers with thread bulbs ( 5 EA's) with one Raspberry Pi 4 - "A" and thread plugs ( 3 EA's ) using another Raspberry Pi 4 - "B" . When  Raspberry Pi 4 - "B" is powered off how can all devices connected to "B" connect to "A" when plugs are in range ? Is there handover mechanism or way for orphan end devices to join "A" and controlled using "A" controller ?

**Comment by jwhui** [comment]
Thread networks are defined by the Operational Dataset they are configured with, including the Network Key. Different Thread networks have different Network Keys. Thread today does not provide a standard mechanism for devices from one Thread network to automatically migrate to another Thread network.

If you want the devices to operate as a single Thread network, you should configure them with the same Operational Dataset.
